demo/demo.md

```diff
@@ -32,10 +32,20 @@ The dropdown can be closed with the following actions:
     - `spacebar`
     - `esc`
 
+## Accessibility requirements
+
+All instances of `auro-dropdown` must have a valid `aria-label` or `aria-labelledby` attribute. There are three options available to meet this requirement. Only one of these options should be implemented.
```

Comment:
This is incorrect - the label slot is also an option, and doesn't require the dev to set aria-label or aria-labelledby themselves. Aria attributes should only be used when absolutely necessary. Can we re-word this to suggest the label slot as the primary method of labelling the dropdown? 

I'd also recommend updating the examples to use the label slot instead of aria-label.

Reply:
@geoffrich The way I wrote this is meant to say that one way or another you must get an `aria-label` or `aria-labelledby` attribute on the trigger. Using the slot is just another means of making that end goal happen.

Regarding if the slot should be our default example or not - I will bring this up during our standup discussion today with @leeejune. I'd like design to weigh in here. I had understood the label slot to be optional but not the default implementation choice.

Reply:
Ah, gotcha - we can go with whatever design says on that front. I still think this is worded confusingly; the user of the component shouldn't care that the label slot sets aria-labelledby behind the scenes.

Reply:
Better, but I think we can make it even simpler. How about "All instances of `auro-dropdown` must have a valid label." for the first sentence? Then the rest is good